docs/Developers/Development_Setup.rst

```diff
@@ -91,25 +91,21 @@ https://gitlab.com/coala/coala-utils/
 Installing from Git
 -------------------
 
-We recommend first installing the latest development snapshot of coala's
-master branch from and all of its dependencies with pip3 using
+We recommend installing coala from the master branch for latest updates
+and its dependencies with pip3 using
```

Comment:
use `pip`, not `pip3` here (Windows installs do not have pip3... but we have another bug about that :/)